modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1639,10 +1639,6 @@
 			<description name="Replace constants of CommerceAccountPortletKeys by constants of AccountPortletKeys" />
 			<weight>2</weight>
 		</check>
-		<check name="UpgradeJavaAddAddressMethodCheck">
-			<category name="Upgrade" />
-			<description name="Fill the new parameter of the method 'addAddress' of AddressLocalService, AddressLocalServiceUtil, AddressService and AddressServiceUtil" />
-		</check>
```

Comment:
@TamyrisTorres, please rename this commit message to 'Delete UpgradeJavaAddAddressMethodCheck unnecessary code' and move this commit to after the addition on UpgradeCatchAllCheck patterns and tests.